ui/quick_query_panel.py

```diff
@@ -201,3 +212,36 @@ def show_query(self):
             self.dialog.text_query.setPlainText(query)
             item = self.dialog.menu_widget.item(self.dialog.query_menu_index)
             self.dialog.menu_widget.setCurrentItem(item)
+
+            
+    def update_friendly(self):
+        """ Updates the QuickQuery Friendly Label (label_qq_friendly) """
+        try:
+            p = self.gather_values()
+        except QuickOsmException as e:
+            self.dialog.display_quickosm_exception(e)
+            return
+        except Exception as e:
+            self.dialog.display_critical_exception(e)
+            return
+
+        # Make the query, in order to create the friendly message
+        query_factory = QueryFactory(
+            query_type=p['query_type'],
+            key=p['key'],
+            value=p['value'],
+            area=p['place'],
+            around_distance=p['distance'],
+            osm_objects=p['osm_objects'],
+            timeout=p['timeout']
+        )
+        try:
+            msg = query_factory.friendly_message()
+        except QuickOsmException as e:
+            self.dialog.display_quickosm_exception(e)
+            self.dialog.label_qq_friendly.setText("")
```

Comment:
```suggestion
            self.dialog.label_qq_friendly.setText(e.message)
```
Would that work?
and removing the `display_quickosm_exception`?

Reply:
Should do - it removes the error from the "alert" banner along the top though - is that what you want?  If so, I'll check it all back in.

Reply:
Yes, the "alert" banner is too intrusive because you will get alert every-time you start typing something, right?